api/src/pcapi/core/offers/api.py

```diff
@@ -378,9 +378,15 @@ def update_educational_offer(
 
 
 def batch_update_offers(query, update_fields):
-    offer_ids_tuples = query.filter(Offer.validation == OfferValidationStatus.APPROVED).with_entities(Offer.id)
+    offer_ids, venue_ids = zip(
+        *query.filter(Offer.validation == OfferValidationStatus.APPROVED).with_entities(Offer.id, Offer.venueId)
+    )
+    venue_ids = sorted(list(set(venue_ids)))
+    logger.info(
+        "Batch update of offers",
+        extra={"updated_fields": update_fields, "nbOffers": len(offer_ids), "venue_ids": venue_ids},
```

Comment:
(détail) pour être homogène avec la casse des autres arguments : 

```suggestion
        extra={"updated_fields": update_fields, "nb_offers": len(offer_ids), "venue_ids": venue_ids},
```